tests/src/test/scala/org/apache/openwhisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -305,6 +305,35 @@ class WskWebActionsTests extends TestHelpers with WskTestHelpers with RestUtil w
       response.body.asByteArray shouldBe Base64.getDecoder().decode(png)
   }
 
+  /**
+   * Tests web action for HEAD requests
+   */
+  it should "create a web action making a HEAD request" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
+    val name = "webactionHead"
+    val file = Some(TestUtils.getTestActionFilename("echo-web-http-head.js"))
+    val host = getServiceURL()
+    val url = s"$host$testRoutePath/$namespace/default/$name"
+
+    assetHelper.withCleaner(wsk.action, name) { (action, _) =>
+      action.create(name, file, web = Some("true"), annotations = Map("require-whisk-auth" -> true.toJson))
+    }
+
+    val unauthorizedResponse = RestAssured.given().config(sslconfig).head(url)
+    unauthorizedResponse.statusCode shouldBe 401
```

Comment:
and then remove this unless this is important to the test.

Reply:
It is as a pattern also used for the other web action tests above. Hence, I included it. The behaviour is not explicit for head or get request so I can remove it